README.md

```diff
@@ -1 +1,15 @@
-# readme take two
+# Stellarpoint
+
+The Stellarpoint Project aims to build a Pose 3D viewer
```

Comment:
The StellarPoint Porject aims to build a 3D viewer can be able to visualize different kinds of sensor data in 3D, e.g. the pose trajectory, the point cloud, the imu data, etc.
We hope to create a web-based visualization tool that can efficiently and accurately display sensor data that is visually appealing and user interactive. It shows the ability to use open source libraries to build a web application to visualize data and convenient debug.